test/unit/submission_collector_test.rb

```diff
@@ -24,7 +24,7 @@ def setup_collector
     @priority_queue = @submission_collector.grouping_queues.find_by_priority_queue(true).groupings
     @regular_queue = @submission_collector.grouping_queues.find_by_priority_queue(false).groupings
     @groupings = []
-    (1..3).each do |i|
+    (1..4).each do |i|
```

Comment:
Unused block argument - `i`. You can omit the argument if you don't care about it.